twill11/hometask-2/run_cli.rb

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require './get_middle_character'
```

Comment:
When I try to run this file from a root folder I see an error message

```
vital@vital-HP-ProBook-450-G7:~/dev/mentor/twill/Ruby-Lab$ ./twill11/hometask-2/run_cli.rb 
Traceback (most recent call last):
        2: from ./twill11/hometask-2/run_cli.rb:4:in `<main>'
        1: from /home/vital/.rvm/rubies/ruby-2.7.4/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:83:in `require'
/home/vital/.rvm/rubies/ruby-2.7.4/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:83:in `require': cannot load such file -- ./get_middle_character (LoadError)
```
```suggestion
require_relative 'get_middle_character'
```